### PR TITLE
more conditions

### PR DESCRIPTION
**Pull request by johannbarbie**

### CODE REVIEW

File: truffle-config.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = {
       settings: {
         optimizer: {
           enabled: true,
-          runs: 200,
+          runs: 2,
```
Comment (troggy) [inline truffle-config.js]: why the change?

Comment (johannbarbie) [reply truffle-config.js]: ah, that was to optimize for deployment size (good for spending conditions). but shouldn't affect the contracts in general.
